Sources/SourceControl/RepositoryManager.swift

```diff
@@ -510,6 +527,12 @@ extension RepositoryManager {
     }
 }
 
+public enum RepositoryUpdateStrategy {
```

Comment:
Is there no way to de-duplicate this with `ContainerUpdateStrategy`? If not, a doc comment here could clarify the duplication.

Reply:
yes because of module separation